_pages/install/browser-support.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Web Browser Support
+---
+
+Ideally, your Spartacus storefront application will render uniformly in any browser that is supported by Spartacus. However, web browsers do not all support the same web standards, nor do they support the same standards in the same way. For example, there are different ways of interpreting a standard's specifications, deliberate design decisions that result in different outcomes, as well as bugs that are unique to each browser. And for each browser that is supported, there is a resulting implementation effort that includes QA, architectural decisions, and technical decisions. In the end, a balance has to be struck between providing an excellent storefront experience, and supporting all the different web browsers that are out there.
+
+Spartacus libraries are designed to take advantage of the latest web platform standards, while also allowing you to run your Spartacus storefront in as many different web browsers as possible. However, some older browsers do not support the latest standards, and as a result, these browsers are not supported by Spartacus. Officially, Spartacus only supports evergreen browsers. This is because Spartacus follows standard Angular, the libraries are kept as clean as possible by intentionally avoiding "browser quirks" in the library code, and there is no planned investment in automated tests of very old browsers.
+
+The approach to browser support in Spartacus is as follows:
+
+- Angular is trusted to be compatible with evergreen browsers.
+- Other standards are trusted to be compatible with evergreen browsers (the [Can I Use?](http://caniuse.com/) website is a good example).
+- Spartacus has many automated quality checks that align the code with those standards (such as the TypeScript standard, and [Can I Use?](http://caniuse.com/)).
+- Spartacus end-to-end tests explicitly test only the Chrome browser.
+
+Nonetheless, with a bit of effort, it is still possible to get Spartacus to work with older browsers. The following are some of the steps you can take to get an older browsers to work with Spartacus:
+
+- Transpile the TypeScript into older versions of JavaScript.
+- Use polyfills to add JavaScript files that can provide some of the standard web features that are missing in some older browsers.
+- Use PostCSS to automatically add vendor prefix style rules when needed in an after process.
+
+## Working with IE11
+
+If your Spartacus storefront needs to support IE11, you must do the following:
```

Comment:
..., you should consider the following steps. Depending on your customisations, there might be other things to consider. For example, if you leverage third party libraries that are not IE11 proof.

Reply:
Please verify if the wording I used is okay. I put most of the details from this comment into a note after the bullet points.